projects/widgets/docs/getting-started.md

```diff
@@ -1,3 +1,48 @@
-# getting started
+# Getting started
 
-...
+Once you have created your widget project on [Projectaanvraag](https://projectaanvraag.uitdatabank.be), you're all set to start building your widget.
+
+Below we explain the different steps of the configuration process in a nutshell.
```

Comment:
This seems more like the "configuration" guide than the "getting started" guide? Getting started should probably explain that you can create your project on projectaanvraag, and "configuration" can cover this "configuration process" that you mention here?

I'll review the rest later once this is cleared up to avoid potentially needless comments.

Reply:
Clear now!